### ISSUE TITLE
babette::update_babette() throws an error on windows

### ISSUE THREAD

**Issue by Giappo** [ropensci/babette#84]
If I try to update babette I obtain the following:

babette::update_babette()
Error in i.p(...) : 
  (contertito da avviso) installation of package ‘C:/Users/p274829/AppData/Local/Temp/RtmpU3tbfH/file4274468377e/tracerer_2.0.4.tar.gz’ had non-zero exit status

Do you know how come this is happening?

**Comment by Giappo** [comment]
Additional info:
1) When I try to build razzo documentation I receive this:
Warning message:
In (function (dep_name, dep_ver = "*")  :
  Need tracerer == 2.0.2 but loaded version is 2.0.1
So apparently I cannot update tracerer to the latest version
2) I am using the microsoft R open version (https://mran.microsoft.com/) which allows me to update up to 3.5.3. I checked on Pedro's machine which runs regular R and 2.0.2 is installed if you call install.packages("tracerer"). I will try to install it from github

**Comment by richelbilderbeek** [comment]
> Do you know how come this is happening?

Sadly: no. I will test on AppVeyor. Thanks for spotting this!

What may help:

```
install.packages("babette")
```

because babette is on CRAN, it will update all deps as well.


> So apparently I cannot update tracerer to the latest version

Hmm, I'd rephrase at 'I really need to update tracerer to the latest version'.

Again: thanks for spotting this. I will add this to AppVeyor before Saturday February 1st 23:59.

**Comment by richelbilderbeek** [comment]
On Monday 27th Jan we observed that some packages were out of date. Updating the packages fixed many things. I predict that this was the cause of this error, so I need do nothing :rainbow: 

Done :+1:
